common/types.go

```diff
@@ -257,7 +259,8 @@ type StackType string
 
 // List of valid stack types
 const (
-	StackTypeVpc          StackType = "vpc"
+	StackTypeAll          StackType = "*"
```

Comment:
this is intended to be an enum of values that an be applied as tags to the stack.  `*` shouldn't be a valid value

Reply:
Understood. I'll come up with a way that doesn't require a special value.

Reply:
Fixed in f6b13e.  I removed the special '*' type, and made a separate func to ListAllStacks(), same as ListStack(stackType), but without the type filter.